# Goal
geometry_msgs/Pose goal_pose
string target_map
---
# Result
bool success
string message
---
# Feedback
float32 progress
